scripts/python3.8/linux/build.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+#
+# This scripts builds a Cura AppImage using a docker container. The docker base
+# image it uses is "cura-build-env:centos7", which can be created with the
+# cura-build-environment repository. The AppImage will be created under the
+# directory "appimages", which will be created if it's not present. Note that
+# the AppImage being created will not be signed.
+#
+
+set -e
+
+# Get where this script resides
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+ROOT_DIR="${SCRIPT_DIR}/../../.."
+
+# Cura release configurations
+CURA_BUILD_ENV_DOCKER_IMAGE="${CURA_BUILD_ENV_DOCKER_IMAGE:-ultimaker/cura-build-environment:centos7-python3.8-qt5.15.2}"
+
+CURA_BRANCH_OR_TAG="${CURA_BRANCH_OR_TAG:-master}"
+URANIUM_BRANCH_OR_TAG="${URANIUM_BRANCH_OR_TAG:-master}"
+CURAENGINE_BRANCH_OR_TAG="${CURAENGINE_BRANCH_OR_TAG:-master}"
+CURABINARYDATA_BRANCH_OR_TAG="${CURABINARYDATA_BRANCH_OR_TAG:-master}"
+FDMMATERIALS_DATA_BRANCH_OR_TAG="${FDMMATERIALS_BRANCH_OR_TAG:-master}"
+LIBCHARON_BRANCH_OR_TAG="${LIBCHARON_BRANCH_OR_TAG:-master}"
+
+CURA_VERSION_MAJOR="${CURA_VERSION_MAJOR:-9}"
```

Comment:
By convention we normally take version 0.0.0 as default if not specified otherwise. Version 9.9.9 is a bit weird, especially if Cura 9 is eventually released.

Reply:
Yes, I agree. Should be 0.0.0